features/support/xmlrpc_system.rb

```diff
@@ -40,4 +40,9 @@ def createSystemRecord(sys_name, ks_label, ip, mac)
     netdevs = [netdev]
     @connection.call('system.createSystemRecord', @sid, sys_name, ks_label, '', '', netdevs)
   end
+
+  # Bootstrap a system (regular salt minion)
```

Comment:
the comment should be, bootstrap a a minion (salt or salt-ssh),

Additional, if you want you could document the new function  yard like:

http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md

This is on todo list https://github.com/SUSE/spacewalk-testsuite-base/issues/373